idea_town/experiments/views.py

```diff
@@ -14,7 +14,7 @@ class ExperimentViewSet(viewsets.ModelViewSet):
     Returns a list of all Experiments in the system.
     """
     queryset = Experiment.objects.all()
-    serializer_class = ExperimentSerializer
+    serializer_class = ExperimentDeepSerializer
```

Comment:
I mainly get antsy when I see an API resource potentially return a lot of data with a lot of DB queries without pagination. It's the kind of thing that's easy to forget about later when the site starts looking hammered.

But, like I said in a later comment, I'm fine with going with this for now and queue up a refinement for later because there really won't be much data at first or even ever (relatively speakin)